catalog/spatial/registry/registry-admin-modules/registry-admin-remote-ui/src/main/webapp/main.js

```diff
@@ -0,0 +1,152 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+/*global require, window */
+/*jslint nomen:false, -W064 */
+
+(function () {
+    'use strict';
+
+    require.config({
+
+        paths: {
+
+            bootstrap: '../../../admin/lib/components-bootstrap/js/bootstrap.min',
+            spin: '../../../admin/lib/spin.js/spin',
+            q: '../../../admin/lib/q/q',
+
+            // backbone
+            backbone: '../../../admin/lib/components-backbone/backbone-min',
+            backbonerelational: '../../../admin/lib/backbone-relational/backbone-relational',
+            backboneassociation: '../../../admin/lib/backbone-associations/backbone-associations-min',
+            underscore: '../../../admin/lib/lodash/dist/lodash.underscore.min',
+            marionette: '../../../admin/lib/marionette/lib/backbone.marionette.min',
+            modelbinder: '../../../admin/lib/backbone.modelbinder/Backbone.ModelBinder.min',
+            collectionbinder: '../../../admin/lib/backbone.modelbinder/Backbone.CollectionBinder.min',
+            poller: '../../../admin/lib/backbone-poller/backbone.poller',
+            iframeresizer: '../../../admin/lib/iframe-resizer/js/iframeResizer.min',
+
+            // ddf
+            spinnerConfig: '../js/spinnerConfig',
+
+            // jquery
+            jquery: '../../../admin/lib/jquery/jquery.min',
+            jqueryui: '../../../admin/lib/jquery-ui/ui/minified/jquery-ui.min',
+            'jquery.ui.widget': '../../../admin/lib/jquery-ui/ui/minified/jquery.ui.widget.min',
+            multiselect: '../../../admin/lib/bootstrap-multiselect/js/bootstrap-multiselect',
+            perfectscrollbar: '../../../admin/lib/perfect-scrollbar/min/perfect-scrollbar-0.4.8.with-mousewheel.min',
+            fileupload: '../../../admin/lib/jquery-file-upload/js/jquery.fileupload',
```

Comment:
Are you actually using the file upload stuff? Seems like there are several dependencies in here that aren't actually being used.